app/views/data_lists/_form.html.erb

```diff
@@ -60,14 +60,16 @@
             <div class="field">
               <%= f2.label :measurements %><br>
               <%= f2.collection_select :measurement_ids, Measurement.where(universe: 'UHERO').all.order(:prefix, :data_portal_name), :id, :prefix_and_name, {:include_blank => false, selected: nil} %>
```

Comment:
This wont let us search over DBEDTCOH measurements to add them to a COH data list.